basketball_reference_web_scraper/data.py

```diff
@@ -240,6 +329,10 @@ def offensive_rebounds(self):
     @property
     def defensive_rebounds(self):
         return self.totals.defensive_rebounds
+    
+    @property
+    def total_rebounds(self):
```

Comment:
I intentionally did not return this field because the data itself can be deduced from the combination of `offensive_rebounds` and `defensive_rebounds` fields.

I appreciate trying to make things easier for the API user, but I also feel this field is unnecessary and increases the total payload returned by the library (obviously, ever so slightly).

Are you adding it because it's solving a pain point for you?

Reply:
Even though this does use up slightly more resources, I added total rebounds because Offensive and defensive rebounds weren't recorded in the NBA until the `1974` season.

Reply:
👍 fair enough - makes sense to me